srcpkgs/dotnet-sdk/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'dotnet-sdk'
+pkgname=dotnet-sdk
+version=5.0.101
+revision=1
+archs="x86_64 aarch64 x86_64-musl"
+create_wrksrc=yes
+short_desc="SDK for making dotnet CORE apps"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="MIT"
+homepage="https://dotnet.microsoft.com"
+
+_target=${DESTDIR}/opt/dotnet
```

Comment:
Hm, do we really want it in `/opt/dotnet`? I don't think we generally have packages in `/opt`. How about `/usr/libexec/dotnet`?

Reply:
While packages generally don't go in `/opt`, given all the various things that the dotnet folder contains and does, I don't know if `/usr/libexec` is a better place. Though, I'm not very experienced and am probably wrong. I'd love to hear your thoughts on this. Thanks ~

Reply:
keeping `/opt` is fine given this is going to be a restricted package from binaries

Reply:
Sounds good